plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/OpenShiftConnector.java

```diff
@@ -195,10 +203,25 @@ public ContainerInfo inspectContainer(String container) throws IOException {
             return null;
         }
 
-        IPod pod = getChePodByContainerId(info.getId());
-        String deploymentConfig = pod.getLabels().get("deploymentConfig");
-        IService svc = getCheServiceBySelector("deploymentConfig", deploymentConfig);
-        Map<String, String> annotations = convertKubernetesNamesToLabels(svc.getAnnotations());
+        Pod pod = getChePodByContainerId(info.getId());
+        if (pod == null ) {
```

Comment:
spaces for if () are different in the file

expected formatting is 
`if (var != value)`

we have
`if ( var != value )`
or
`if (var != value )`
or
`if (var!=value)`